frontend/src/components/app/App.jsx

```diff
@@ -59,7 +77,11 @@ export default function App() {
 
             {/* Left Hamburger Menu */}
             <Route exact path="/users" render={(props) => <UsersPage {...props} />} />
+
             <Route exact path="/sample_types" render={(props) => <SampleTypesPage {...props} />} />
+            <Route exact path="/sample_types/new" render={(props) => <SampleTypeDefinitionForm {...props} />} />
+            <Route exact path="/sample_types/:id/edit" render={(props) => <SampleTypeDefinitionForm {...props} />} />
+
```

Comment:
After all the <Route... /> elements we should redirect any other paths somewhere.  Right now the system just puts up a blank page with a header, which is really weird. (This is the closest place I could add this comment.)